backend/src/handlers/category_operations.py

```
@@ -13,7 +13,7 @@
 from models.transaction import Transaction
 from services.category_rule_engine import CategoryRuleEngine
 from utils.db_utils import create_category_in_db, delete_category_from_db, get_category_by_id_from_db, list_categories_by_user_from_db, update_category_in_db, list_user_transactions
-from utils.lambda_utils import mandatory_path_parameter, optional_query_parameter
+from utils.lambda_utils import mandatory_path_parameter, optional_query_parameter, mandatory_body_parameter, optional_body_parameter, mandatory_query_parameter
```

Ruff (0.11.9)

`utils.lambda_utils.mandatory_query_parameter` imported but unused

Remove unused import: `utils.lambda_utils.mandatory_query_parameter`

(F401)